### PR TITLE
VIVI-8641 Only resolve local IPs

### PR DESCRIPTION
**Pull request by deanylev**

### CODE REVIEW

File: bin/xip-pdns [hunk 1]
```diff
@@ -91,8 +93,8 @@ subdomain_is_ns() {
   [[ "$SUBDOMAIN" =~ $NS_SUBDOMAIN_PATTERN ]]
 }
 
-subdomain_is_ip() {
-  [[ "$SUBDOMAIN" =~ $IP_SUBDOMAIN_PATTERN ]]
```
Comment (tristanpenman) [inline bin/xip-pdns:95]: Do we still want to check the `IP_SUBDOMAIN_PATTERN` pattern?

Comment (deanylev) [reply bin/xip-pdns:95]: So like `[[ "$SUBDOMAIN" =~ $IP_SUBDOMAIN_PATTERN && "$SUBDOMAIN" =~ $LOCAL_IP_SUBDOMAIN_PATTERN ]]`?

Comment (tristanpenman) [reply bin/xip-pdns:95]: yep

Comment (deanylev) [reply bin/xip-pdns:95]: Done

**Comment by deanylev** [comment]
Have tested this and it works, will deploy next week

**Comment by tristanpenman** [comment]
Let's save this for a morning deployment

**Review by tristanpenman — APPROVED** [review]

**Pull request merged**